release-v1.4/rollen-tekst-en-opmaak (#198)
MinBZK/Algoritmekader: docs/rollen/index.md

---
title: Rollen
summary: Hulp bij het bepalen van rollen en verantwoordelijkheden om als overheid verantwoordelijk om te gaan met algoritmes en AI.
icon: material/account
---
# Rollen

> Verantwoorde algoritmes en AI-systemen zijn mogelijk als mensen in verschillende rollen goed samenwerken, op het juiste moment.

Reviewer: ```suggestion
Verantwoord gebruik van algoritmes en AI-systemen zijn mogelijk als mensen in verschillende rollen goed samenwerken, op het juiste moment.
```

Author: ```suggestion
Verantwoord gebruik van algoritmes en AI-systemen is mogelijk als mensen in verschillende rollen goed samenwerken, op het juiste moment.
```

## Bepaal zelf rollen en verantwoordelijkheden
Organisaties bepalen zelf de namen van hun rollen of functies en de taken en verantwoordelijkheden die hierbij horen. 
Sommige organisaties hebben bijvoorbeeld minder rollen, omdat zij verschillende expertises combineren in één rol. Of rollen hebben andere namen.

> ## Voorbeelden van rollen

Reviewer: Ik vraag me af of je hier ook een opsomming wilt? Er is ook al een menu aan de linkerkant van de pagina?

Author: Graag linkermenu weghalen.

Reviewer: voor nu gekozen om het linkermenu voor nu te laten staan. Dat wordt automatisch gemaakt. Gezien de rollen nog veranderlijk zijn, heeft dat de voorkeur. Een handmatige lijst zorgt ervoor dat rollen op twee plekken moeten worden toegevoegd. Dat is foutgevoelig. Wanneer we overgaan tot een andere huisstijl kan dit mogelijk op een andere manier opgelost worden.

> De volgende rollen passen bij de manier van werken van grote organisaties zoals overheden. Andere rollen zijn ook mogelijk:
> - [Aanbestedingsjurist](https://minbzk.github.io/Algoritmekader/rollen/aanbestedingsjurist/)

Reviewer: ```suggestion
De volgende rollen passen bij de manier van werken van grote organisaties zoals overheden. Andere rollen zijn ook mogelijk:

- [Aanbestedingsjurist](https://minbzk.github.io/Algoritmekader/rollen/aanbestedingsjurist/)
```

> De volgende rollen passen bij de manier van werken van grote organisaties zoals overheden. Andere rollen zijn ook mogelijk:
> - [Aanbestedingsjurist](aanbestedingsjurist.md)
> - [Aanbieder](aanbieder.md)
> - [Archiefdeskundige](archiefdeskundige.md)
> - [Behoeftesteller](behoeftesteller.md)
> - [Beleidsmedewerker](beleidsmedewerker.md)
> - [Communicatieadviseur](communicatieadviseur.md)
> - [Contractbeheerder](hcontractbeheerder.md)
> - [Data engineer](data-engineer.md)
> - [Data scientist](data-scientist.md)
> - [Domeinspecialist](domeinspecialist.md)
> - [Ethicus](ethicus.md)
> - [Gebruiker](gebruiker.md)
> - [Gemandateerd verantwoordelijke](gemandateerd-verantwoordelijke.md)
> - [Informatie analist](informatie-analist.md)
> - [Informatiebeheerder](informatiebeheerder.md)
> - [Inkoopadviseur](inkoopadviseur.md)
> - [Jurist](jurist.md)
> - [Opdrachtgever](opdrachtgever.md)
> - [Opdrachtnemer](opdrachtnemer.md)
> - [Privacy-officer](privacy-officer.md)
> - [Proceseigenaar](proceseigenaar.md)
> - [Projectleider](projectleider.md)
> - [Security officer](security-officer.md)

Reviewer: ```suggestion
De rollen in het linkermenu spelen in het ideale geval een rol bij het gebruik van algoritmes. Andere rollen zijn ook mogelijk.
```

> - [Aanbestedingsjurist](https://minbzk.github.io/Algoritmekader/rollen/aanbestedingsjurist/)
> - [Aanbieder](https://minbzk.github.io/Algoritmekader/rollen/aanbieder/)
> - [Archiefdeskundige](https://minbzk.github.io/Algoritmekader/rollen/archiefdeskundige/)
> - [Behoeftesteller](https://minbzk.github.io/Algoritmekader/rollen/behoeftesteller/)
> - [Beleidsmedewerker](https://minbzk.github.io/Algoritmekader/rollen/beleidsmedewerker/)
> - [Communicatieadviseur](https://minbzk.github.io/Algoritmekader/rollen/communicatieadviseur/)
> - [Contractbeheerder](https://minbzk.github.io/Algoritmekader/rollen/contractbeheerder/)
> - [Data engineer](https://minbzk.github.io/Algoritmekader/rollen/data-engineer/)
> - [Data scientist](https://minbzk.github.io/Algoritmekader/rollen/data-scientist/)
> - [Domeinspecialist](https://minbzk.github.io/Algoritmekader/rollen/domeinspecialist/)
> - [Ethicus](https://minbzk.github.io/Algoritmekader/rollen/ethicus/)
> - [Gebruiker](https://minbzk.github.io/Algoritmekader/rollen/gebruiker/)
> - [Gemandateerd verantwoordelijke](https://minbzk.github.io/Algoritmekader/rollen/gemandateerd-verantwoordelijke/)
> - [Informatie analist](https://minbzk.github.io/Algoritmekader/rollen/informatie-analist/)
> - [Informatiebeheerder](https://minbzk.github.io/Algoritmekader/rollen/informatiebeheerder/)
> - [Inkoopadviseur](https://minbzk.github.io/Algoritmekader/rollen/inkoopadviseur/)
> - [Jurist](https://minbzk.github.io/Algoritmekader/rollen/jurist/)
> - [Opdrachtgever](https://minbzk.github.io/Algoritmekader/rollen/opdrachtgever/)
> - [Opdrachtnemer](https://minbzk.github.io/Algoritmekader/rollen/opdrachtnemer/)
> - [Privacy-officer](https://minbzk.github.io/Algoritmekader/rollen/privacy-officer/)
> - [Proceseigenaar](https://minbzk.github.io/Algoritmekader/rollen/proceseigenaar/)
> - [Projectleider](https://minbzk.github.io/Algoritmekader/rollen/projectleider/)
> - [Security officer](https://minbzk.github.io/Algoritmekader/rollen/security-officer/)

Reviewer: ```suggestion
- [Aanbestedingsjurist](aanbestedingsjurist.md)
- [Aanbieder](aanbieder.md)
- [Archiefdeskundige](archiefdeskundige.md)
- [Behoeftesteller](behoeftesteller.md)
- [Beleidsmedewerker](beleidsmedewerker.md)
- [Communicatieadviseur](communicatieadviseur.md)
- [Contractbeheerder](hcontractbeheerder.md)
- [Data engineer](data-engineer.md)
- [Data scientist](data-scientist.md)
- [Domeinspecialist](domeinspecialist.md)
- [Ethicus](ethicus.md)
- [Gebruiker](gebruiker.md)
- [Gemandateerd verantwoordelijke](gemandateerd-verantwoordelijke.md)
- [Informatie analist](informatie-analist.md)
- [Informatiebeheerder](informatiebeheerder.md)
- [Inkoopadviseur](inkoopadviseur.md)
- [Jurist](jurist.md)
- [Opdrachtgever](opdrachtgever.md)
- [Opdrachtnemer](opdrachtnemer.md)
- [Privacy-officer](privacy-officer.md)
- [Proceseigenaar](proceseigenaar.md)
- [Projectleider](projectleider.md)
- [Security officer](security-officer.md)
```
